src/models/d2sb/SBDraft2CommandLineBindingModel.ts

```diff
@@ -10,6 +10,7 @@ export class SBDraft2CommandLineBindingModel extends CommandLineBindingModel imp
     public valueFrom: SBDraft2ExpressionModel;
     public hasSecondaryFiles = true;
     public hasShellQuote = false;
+    public itemSeparator: string = "null";
```

Comment:
Did it had to be 'null' as a string :(

Reply:
@marijanlekic Maybe we should create constant for this, something like 
`const EXECUTOR_REPEAT_PREFIX = "null"`

And export it so we can use it in Kasina